span.go

```diff
@@ -49,7 +49,7 @@ func (e *traceExporter) convertSpan(s *trace.SpanData) *ddSpan {
 	span := &ddSpan{
 		TraceID:  binary.BigEndian.Uint64(s.SpanContext.TraceID[8:]),
 		SpanID:   binary.BigEndian.Uint64(s.SpanContext.SpanID[:]),
-		Name:     s.Name,
+		Name:     "opencensus",
```

Comment:
do we have alternatives for the default name?

Reply:
We’ve given this some thought and couldn’t find other options. This will look acceptable in the UI as a generic solution. It will be “opencensus /resource/path” on spans. 

Users can still change it if they will, using “span.name” tag.